_i18n/ar/resources/moneropedia/merge-mining.md

```diff
@@ -0,0 +1,20 @@
+---
+entry: "Merge Mining"
+terms: ["merge-mine", "merge-mining", "merged-mining","merge-mined"]
+summary: "the process of mining two or more blockchains at the same time "
+---
+
+{% include disclaimer.html translated="no" translationOutdated="no" %}
+
+### The Basics
+
+Merge mining is the process of @mining two or more @blockchains at the same time. Generally there is a parent @blockchain and an auxillary blockchain both of which use the same proof of work algorithm. In most cases the auxillary chain will have a lower difficulty than the parent block chain.
+
+Merge mining can be used for securing a lower hashrate blockchain or developing a @sidechain that does something different than the primary chain.
+Some examples that currently exist or are in development are: a blockchain game (TownForge), digital assets platform (Tari) or a decentralized mining pool (@P2Pool).
+
+#### External Links
+
+- [Tari - What is Tari](https://www.tari.com/#what-is-tari)
+- [Townforge -About](https://townforge.net/about)
+- [P2Pool GitHub](https://townforge.net/about)
```

Comment:
I don't think it's necessary to have a dedicated section for external links. Maybe we could add them directly in the paragraph where we mention them? We also don't need to add a link to p2pool website if we link to the Moneropedia entry.

```suggestion
Merge mining can be used for securing a lower hashrate blockchain or developing a @sidechain that does something different than the primary chain.
Some examples that currently exist or are in development are: a blockchain game ([TownForge](https://townforge.net/about)), digital assets platform ([Tari](https://www.tari.com/#what-is-tari)) or a decentralized mining pool (@P2Pool).
```